packages/merge/src/modules/merge/sagas.spec.js

```diff
@@ -55,23 +68,36 @@ describe('merge', () => {
         describe('executeMerge', () => {
           test('should call executeMerge', () => {
             const body = {
-              selection: selection,
+              selection: expectedSelection,
               targetEntity: {
                 key: '1',
                 paths: {}
               },
               mergeRelations: []
             }
             const response = {
-              success: true
+              notCopiedRelations: [
```

Comment:
is there a test that this line 
https://github.com/tocco/tocco-client/pull/1236/files#diff-bedba0a7c6df678291a8c34ff4d281b65eb1442e06adc222aedfd9d001fd6d00R47
with false?

Reply:
in the test getMergeBody the callname has writable=false, so this line is already tested